sim/ofat.py

```diff
@@ -4,6 +4,11 @@
 import glob
 import json
 import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy
+from pathlib import Path
+import shutil
```

Comment:
Seem's like you don't use shutil, numpy or mpl?

Reply:
Yeah you were right about shutil and numpy, must have been legacy imports! mpl I think is required for pyplot, although I may be wrong about that

Reply:
I think you don't need it. Try without and see

Reply:
Ok yeah you were right, removed